### ISSUE TITLE
Need to explicitly specify extraneous dependencies in Cargo.toml

### ISSUE THREAD

**Issue by maksyms** [constellation-rs/amadeus#96]
I'm trying to build a new project using `amadeus`. I've copy/pasted the first example from README.md and added the following into Cargo.toml:

```
[dependencies]
amadeus = { version = "0.3.5", features = ["aws", "parquet"] }
tokio = "0.2.22"
```

The project fails to build, complaining that it cannot find `amadeus_core`, `amadeus_types`, `amadeus_parquet`.

The only way I could compile the project is if I explicitly list the extraneous dependencies:

```
[dependencies]
amadeus = { version = "0.3.5", features = ["aws", "parquet"] }
tokio = "0.2.22"
# I don't understand why these need to be explicit, rather than features above
amadeus-core = "0.3.5"
amadeus-types = "0.3.5"
amadeus-parquet = "0.3.5"
```

I'd like to think that amadeus package itself would include at least `core` and `types`, as well as when the extra feature like `parquet` is specified, it would bring in `amadeus_parquet`. What am I missing? Is this a bug?

**Comment by alecmocatta** [comment]
Oh dear that's embarassing. Silly oversight that is fixed in #97.  Thanks for the heads up @maksyms! v0.3.6 with the fix will be published shortly.

**Comment by maksyms** [comment]
No worries, thanks for implementing a fix so quickly, @alecmocatta!
